cmd/oras/internal/display/status/text.go

```diff
@@ -65,9 +66,13 @@ func (ph *TextPushHandler) UpdateCopyOptions(opts *oras.CopyGraphOptions, fetche
 	}
 	opts.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
 		committed.Store(desc.Digest.String(), desc.Annotations[ocispec.AnnotationTitle])
-		if err := output.PrintSuccessorStatus(ctx, desc, fetcher, committed, ph.printer.StatusPrinter(PushPromptSkipped)); err != nil {
+		successors, err := descriptor.GetSuccessors(ctx, desc, fetcher, DeduplicatedFilter(committed))
+		if err != nil {
 			return err
 		}
+		for _, successor := range successors {
+			_ = ph.printer.PrintStatus(successor, PushPromptSkipped)
```

Comment:
Should the return error be checked?

Reply:
All the printer print errors returned will be print to err or stderr and be ignored and thus, no need to handle. Do you think it's a good idean remove error as the returned error like cobra [cmd.Println](https://github.com/spf13/cobra/blob/v1.8.1/command.go#L1403-L1406).

Reply:
Writing to `stdout` or `stderr` can fail in some circumstances. Therefore, we do need to catch those errors. In those cases, printing the error messages may not be goal but interrupting the program to continue to run.

Reply:
You are right. Sometimes even the format output can return error, which is terminal.